index.html

```diff
@@ -4975,6 +4975,637 @@ <h2 id="AudioWorkerParamDescriptor">
+              <dt>
+                ChannelCountMode channelCountMode
+              </dt>
+              <dd>
+                Same to <a>channelCountMode</a> in <a>AudioNode</a>.
+              </dd>
+              <dt>
+                ChannelInterpretation channelInterpretation
+              </dt>
+              <dd>
+                Same to <a>channelInterpretation</a> in <a>AudioNode</a>.
+              </dd>
+            </dl>
+          </section>
+        </section>
+        <section>
+          <h2 id="AudioWorkletGlobalScope">
+            The <dfn>AudioWorkletGlobalScope</dfn> interface
+          </h2>
+          <p>
+            This interface is derived from <code>WorkletGlobalScope</code> and
+            represents a special execution context in which an audio processing
+            script is run. It is designed to enable the generation, processing,
+            and analysis of audio data directly using JavaScript in the audio
+            <a>rendering thread</a>. The user-supplied script code is evaluated
+            in this scope to define an
+            <a><code>AudioWorkletProcessor</code></a>.
+          </p>
+          <dl title="interface AudioWorkletGlobalScope : WorkletGlobalScope"
+          class="idl">
+            <dt>
+              void registerAudioWorkletProcessor (DOMString name, Function
+              classConstructor)
+            </dt>
+            <dd>
+              <p>
+                Registers a definition of a class that is derived from
+                <a>AudioWorkletProcessor</a>.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  DOMString name
+                </dt>
+                <dd>
+                  A string key that represents a class definition to be
+                  registered.
+                </dd>
+                <dt>
+                  Function classConstructor
+                </dt>
+                <dd>
+                  A class definition extended from
+                  <a>AudioWorkletProcessor</a>.
+                </dd>
+              </dl>
+            </dd>
+          </dl>
+        </section>
+        <section>
+          <h2 id="AudioWorkletProcessor">
+            The <dfn>AudioWorkletProcessor</dfn> interface
+          </h2>
+          <p>
+            This interface represents an audio processing module that runs on
+            the audio <a>rendering thread</a>. It is defined in an
+            <a><code>AudioWorkletGlobalScope</code></a> and the definition of
+            the class manifests the actual audio processing mechanism of a
+            custom audio node. <a><code>AudioWorkletProcessor</code></a> can
+            only be instantiated by the construction of an
+            <a><code>AudioWorkletNode</code></a> instance.
+          </p>
+          <dl title="interface AudioWorkletProcessor" class="idl">
+            <dt>
+              readonly attribute static sequence&lt;AudioParamDescriptor&gt;
```

Comment:
Is this really the right way to write this?  What is the name of this attribute?  Should it be something like

`static sequence<AudioParamDescriptor> attributeName`